pandas/io/json/_json.py

```diff
@@ -750,6 +740,8 @@ def read_json(
 }}\
 '
     """
+    if isinstance(path_or_buf, str):
+        raise TypeError("cannot pass literal json to pandas.read_json")
```

Comment:
I think we can only add a warning for now and then with 3.0 (or 2.2?) remove support for json literals.

Reply:
Oh wow so if that's the case, I'll probably want to revert most of my changes. 

It seems like the only addition here would be to display a warning message to the client if they pass a literal string to `read_json` then?

Reply:
Oh wow, that would be drastically different compared to the original ask in the GitHub issue. Though, that would follow along the same lines of what we have seen with `DataFrame.concat` and `DataFrame.append`

Reply:
Although I wrote the bug as `ENH`, my plan (although thanks for picking it up since I didn't get enough roundtoits until now), was to introduce a deprecation warning and then remove in an upcoming release. I think it's too user- facing to just remove immediately.

Reply:
@wence- That makes perfect sense. 

I may hit pause on this until for a day or two until we hear from `mroeschke` or another reviewer for official direction. I don't want to steer too far away from the original acceptance criteria / guidance in the Feature Description, although I think that they'll probably echo what you're suggesting :smile:

Reply:
Yeah @twoertwein and @wence- are correct here. We will need to add a warning to deprecate the current behavior of parsing string json in this PR before we can outright raise an error

Reply:
@mroeschke Awesome! Thanks!